File: examples/source/access_array.cpp
int access_array(int index) {
    int arr[10] {};

    arr[0] = 0;
    arr[1] = 1;
    arr[2] = 2;
    arr[3] = 3;
    arr[4] = 4;
    arr[5] = 5;
    arr[6] = 6;
    arr[7] = 7;
    arr[8] = 8;
    arr[9] = 9;

    return arr[index];
}

int main() {
    int _ret = access_array(10);
}

File: examples/source/access_array.rs
pub fn access_array(index: i32) -> i32 {
    let arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
    arr[index as usize]
}

#[allow(dead_code)]
fn main() {
    let _ret = access_array(10);
}
